Test failure: snapshot_in_svnn_transition_test

https://github.com/AntelopeIO/spring/actions/runs/16178381890/job/45669611321

```
Traceback (most recent call last):
  File "/__w/spring/spring/build/tests/snapshot_in_svnn_transition_test.py", line 128, in <module>
    restartWithSnapshot(nodeSnap, rmBlocks=True, updateChainArg=False)
  File "/__w/spring/spring/build/tests/snapshot_in_svnn_transition_test.py", line 119, in restartWithSnapshot
    assert node.waitForLibToAdvance(), "LIB did not advance after restart from snapshot"
AssertionError: LIB did not advance after restart from snapshot
```

It appears the node was just slow starting up and syncing seems we need a bit longer timeout on the verification that LIB is moving.

--- a/tests/snapshot_in_svnn_transition_test.py
+++ b/tests/snapshot_in_svnn_transition_test.py
@@ -116,7 +116,7 @@ try:
           chainArg=None
        isRelaunchSuccess = node.relaunch(chainArg=chainArg)
        assert isRelaunchSuccess, "Failed to relaunch node with snapshot"
-       assert node.waitForLibToAdvance(), "LIB did not advance after restart from snapshot"
+       assert node.waitForLibToAdvance(timeout=60), "LIB did not advance after restart from snapshot"
 
     Print("Restart snapshot node with snapshot with blocks")
     restartWithSnapshot(nodeSnap, rmBlocks=False)

